<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="singlPlayerPage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="SnakeGameView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>821</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>291</x>
          <y>581</y>
          <width>214</width>
          <height>27</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="home_btn">
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startPushButton">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="MultiPlayerPage" name="multiplayerPage"/>
      <widget class="QWidget" name="aiPage">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="NewMainWindow" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="mainPage">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_ai">
          <property name="text">
           <string>ai page</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QPushButton" name="pushButton_createRoom">
          <property name="text">
           <string>Create Room</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" rowspan="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_singlePlayer">
          <property name="text">
           <string>single Player</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Go to Home Page</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SnakeGameView</class>
   <extends>QGraphicsView</extends>
   <header>snakeGameView/snakegameview.h</header>
  </customwidget>
  <customwidget>
   <class>MultiPlayerPage</class>
   <extends>QWidget</extends>
   <header>multiplayerpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NewMainWindow</class>
   <extends>QWidget</extends>
   <header>newmainwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
